.github/workflows/publish-package-release.yml

```diff
@@ -161,8 +161,8 @@ jobs:
       continue-on-error: true
 
     - name: Update Changelog
-      working-directory: ${{ github.workspace }}/${{ env.PACKAGE_PATH }}
-      run: ${{ github.workspace }}/scripts/changelog-header.sh
+      working-directory: 
```

Comment:
Is there a default when this is not defined? (And is it still needed here?)

Reply:
Good catch, it should have been removed entirely
```suggestion
```